Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -2071,11 +2071,13 @@ void ParseV3000AtomBlock(std::istream *inStream, unsigned int &line,
   }
 
   if (mol->hasProp(common_properties::_2DConf)) {
```

Comment:
The way this logic is structured we don't need the `_2DConf` property at all any more.